clear timeout after reset polling

alonspr opened the pull request:
when resetting a specific polling - clearing it's direct timeout to avoid polling duplications.

Commit:
clear time after reset polling

naorye commented:
Cool. You got a point. In case the polling starts right before the timeout done, there is requests duplication. The same occurs for stop action.
Can you complete it?

naorye commented on src/logic.js:
```diff
@@ -1,6 +1,8 @@
 import { getPollingState } from './reducer';
 
+let timeouts = {};
+
```
Please use const (I suggest using airbnb linter)

alonspr replied:
Fixed !  also added the clear timeout for the stop method.

alonspr committed:
clearing timeout for stop method

- used const instead of let for timeouts.
- clearing timeout for stop method as well.
